.php_cs

```diff
@@ -0,0 +1,63 @@
+<?php
+
+if (class_exists('Symfony\CS\Config\Config')) {
+    // PHP-CS-Fixer 1.x syntax (deprecated)
+    return Symfony\CS\Config\Config::create()
+        ->setUsingLinter(false)
+        ->setUsingCache(true)
+        ->level(Symfony\CS\FixerInterface::SYMFONY_LEVEL)
+        ->fixers([
+            'concat_with_spaces',
+            '-concat_without_spaces',
+            '-empty_return',
+            '-phpdoc_params',
+            '-phpdoc_separation',
+            '-phpdoc_to_comment',
+            '-spaces_cast',
+            '-blankline_after_open_tag',
+            '-single_blank_line_before_namespace',
+            '-phpdoc_annotation_without_dot',
+        ])
+        ->finder(
+            Symfony\CS\Finder\DefaultFinder::create()
+                ->in(__DIR__)
+                ->exclude([
+                    'docs',
+                    'vendor',
+                    'src/bin'
+                ])
+                ->files()->name('*.php')
+        )
+    ;
+}
+
+// PHP-CS-Fixer 2.x syntax
+return PhpCsFixer\Config::create()
+    ->setRules([
+        '@Symfony' => true,
+        '@Symfony:risky' => true,
+        'concat_space' => ['spacing' => 'one'],
+        'array_syntax' => false,
```

Comment:
@andrerom I know you are bored by this ;) but maybe here we can do `'array_syntax' => ['syntax' => 'short'],` and fix everything at once? :D
(composer requires at least PHP 5.4 and we don't have multiple supported branches yet).

Reply:
it's ok on this repo